WatchApp/Base.lproj/Interface.storyboard

```diff
@@ -468,6 +440,88 @@
             </objects>
             <point key="canvasLocation" x="38" y="1153"/>
         </scene>
+        <!--Loop-->
+        <scene sceneID="mYv-uf-Zz0">
+            <objects>
+                <controller identifier="DataHUDController" title="Loop" alwaysBounce="YES" id="L6b-jz-BPv" customClass="DataHUDController" customModule="WatchApp_Extension">
+                    <items>
+                        <group width="1" alignment="left" id="pXN-2b-PyZ">
+                            <items>
+                                <group width="0.25" alignment="left" layout="vertical" id="DSZ-LM-LvX">
+                                    <items>
+                                        <imageView alignment="center" image="loop_unknown" contentMode="center" id="6ud-R0-YBi"/>
+                                    </items>
+                                </group>
+                                <group width="0.75" alignment="left" layout="vertical" id="tbu-dc-kPL">
+                                    <items>
+                                        <group width="1" height="26" alignment="left" id="B98-ff-cNG">
+                                            <items>
+                                                <label height="1" alignment="left" hidden="YES" text="---" textAlignment="center" id="lww-lD-Daq">
+                                                    <fontDescription key="font" type="system" weight="light" pointSize="24"/>
+                                                </label>
+                                                <label height="1" alignment="right" hidden="YES" text="---" textAlignment="center" id="FJh-kf-aL9">
+                                                    <color key="textColor" red="1" green="1" blue="1" alpha="0.80000000000000004" colorSpace="custom" customColorSpace="sRGB"/>
+                                                    <fontDescription key="font" type="system" weight="light" pointSize="24"/>
+                                                </label>
+                                            </items>
+                                            <variation key="device=watch42mm" height="29"/>
+                                        </group>
+                                    </items>
+                                    <variation key="device=watch40mm" widthAdjustment="-8.5"/>
+                                    <variation key="device=watch44mm" widthAdjustment="-9.5"/>
+                                </group>
+                            </items>
+                        </group>
+                        <group width="1" alignment="left" layout="vertical" spacing="0.0" id="seG-UI-feg">
+                            <items>
+                                <label alignment="left" text="INSULIN ON BOARD" id="kvt-sg-hNd">
```

Comment:
Using "Active Insulin" (and below, "Active Carbs") would be more consistent with the iPhone app - plus "on board" is a bit of an odd English idiom that might be a little harder to translate English language learners if they're not running a localized version in another language.